6.0/6.0.1/01-command_line_arguments.md

```diff
@@ -0,0 +1,192 @@
+<!-- Space: TOOLS -->
+<!-- Parent: Installer -->
+<!-- Title: Command Line Arguments -->
+<!-- Layout: plain -->
+
+# Command line arguments explained
+
+<br />
+
+## Phase: `deploy`
+
+`--skip-namespace`
+
+- installer does not deploy the `namespace.yaml` manifest.
+  It expects the Namespace to exist and to match the value in `values.yaml`
+  There is no validation, in case of mismatch the installer will fail
+
+`--skip-pull-secret`
+
+- the services expect the pull secret to exist,
+  to have the expected name (`sysdigcloud-pull-secret`) and to allow access to the registry.
+- if the pull secret is missing, the behaviour could be unpredictable:
+  some Pods could start if they can find the image locally and if their `imagePullPolicy`
+  is not `Always`
+- Other Pods will fail because they can't pull the image
+
+`--skip-serviceaccount`
+
+- The user must provide SAs with the exact same name expected:
+
+```
+sysdig-serviceaccount.yaml:  name: sysdig
+sysdig-serviceaccount.yaml:  name: node-labels-to-files
+sysdig-serviceaccount.yaml:  name: sysdig-with-root
+sysdig-serviceaccount.yaml:  name: sysdig-elasticsearch
+sysdig-serviceaccount.yaml:  name: sysdig-cassandra
+```
+
+- One implication of this is that unless the `node-to-labels` SA is added,
+  rack awareness will not work neither in Cassandra nor in ES (to be verified)
```

Comment:
Does it mean that Elastic Search is not tested?

Reply:
@bennythejudge This information was taken from installer docs, could you confirm what is asked by @radhikapc?

Reply:
Should we call out that ES is not tested?

Reply:
@radhikapc It has nothing to do with that. If someone uses this commandline option in Installer then they have some implications with that action. All default configuration will work as is. This is an advanced configuration setting.